tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -617,10 +617,10 @@ $ bazel build -c opt //tensorflow/tools/pip_package:build_pip_package
 # To build with GPU support:
 $ bazel build -c opt --config=cuda //tensorflow/tools/pip_package:build_pip_package
 
-$ bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg
+$ sudo bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg
```

Comment:
I try to again using "bazel-bin/tensorflow/tools/pip_package/build_pip_package
/tmp/tensorflow_pkg"

A error occurred, I captured that message

IOError: [Errno 13] Permission denied:

'/Library/Python/2.7/site-packages/protobuf-3.0.0b2-py2.7.egg/EGG-INFO/namespace_packages.txt'

because of namespace_package.txt is allow to admin (root)
I change add sudo command "sudo
bazel-bin/tensorflow/tools/pip_package/build_pip_package
/tmp/tensorflow_pkg".

$ ls -al
/Library/Python/2.7/site-packages/protobuf-3.0.0b2-py2.7.egg/EGG-INFO/

total 56

drwxr-xr-x  2 root  wheel   306  4 20 18:13 .

drwxr-xr-x  4 root  wheel   136  4 20 18:13 ..

-rw-r-----  1 root  wheel   662  4 20 18:13 PKG-INFO

-rw-r-----  1 root  wheel  2801  4 20 18:13 SOURCES.txt

-rw-r-----  1 root  wheel     1  4 20 18:13 dependency_links.txt

-rw-r-----  1 root  wheel     7  4 20 18:13 namespace_packages.txt

-rw-r-----  1 root  wheel    19  4 20 18:13 requires.txt

-rw-r-----  1 root  wheel     7  4 20 18:13 top_level.txt

-rw-r--r--  1 root  wheel     1  4 20 18:13 zip-safe


the result of add sudo command, it is no problem. complete progress.
and I also found that "/tmp" is possible to us non-admin user at OS x EL
Capitan version.

How did you think about? it is special case on my environment? or general
case on OS X environment.






On Fri, Apr 29, 2016 at 9:22 AM, Martin Wicke <notifications@github.com>
wrote:

> In tensorflow/g3doc/get_started/os_setup.md
> <https://github.com/tensorflow/tensorflow/pull/2095#discussion_r61522747>:
>
> > @@ -617,10 +617,10 @@ $ bazel build -c opt //tensorflow/tools/pip_package:build_pip_package
> >  # To build with GPU support:
> >  $ bazel build -c opt --config=cuda //tensorflow/tools/pip_package:build_pip_package
> >
> > -$ bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg
> > +$ sudo bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg
>
> /tmp doesn't need sudo on my machine, which is a Mac with El Capitan.
> Maybe it does for non-admin users? Even that would very much surprise me
> though.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/2095/files/7d1417ce568bb5d28b215d7272549751cd599c63#r61522747>
>